alacritty/src/event.rs

```diff
@@ -1327,21 +1339,167 @@ impl Processor {
         }
     }
 
+    /// Create the GL display.
+    fn create_gl_display(
```

Comment:
Questionable these OpenGL functions should be in `alacritty/src/event.rs`? Maybe move to display?